src/process_utility.c

```diff
@@ -132,16 +131,17 @@ check_chunk_alter_table_operation_allowed(Oid relid, AlterTableStmt *stmt)
 				case AT_ClusterOn:
 				case AT_EnableRowSecurity:
 				case AT_DisableRowSecurity:
+				case AT_SetTableSpace:
 					/* allowed on chunks */
+					allowed = true;
 					break;
 				default:
 					/* disable by default */
-					all_allowed = false;
```

Comment:
this needs to remain, see my other comment

Reply:
Oh you right. Sorry, I guess I drunk too much tea